src/functions.php

```diff
@@ -6,7 +6,14 @@
 if (!function_exists('appBasePath')) {
     function appBasePath(): string
```

Comment:
Hello!. 

How about this? 

```php
function appBasePath(): string
{
    $pwd = defined('LARAVEL_START') ? app()->basePath() : getcwd();

    $basePath = rtrim($pwd, DIRECTORY_SEPARATOR) . DIRECTORY_SEPARATOR;

    foreach (['public', 'pub', 'wp-admin'] as $dir) {
        if (str_ends_with($basePath, $dir . DIRECTORY_SEPARATOR)) {
            $basePath = substr($basePath, 0, -strlen($dir . DIRECTORY_SEPARATOR));

            break;
        }
    }

    return $basePath;
}
```

Reply:
Hmm. Plugged it into my Octane env, looks like LARAVEL_START does not get defined. Probably the Octane server does not include the standard index.php where it would be defined

Reply:
I used your code and just changed it to
`$pwd = defined('LARAVEL_START') || isset($_SERVER['LARAVEL_OCTANE']) ? app()->basePath() : getcwd();`

My thoughts are: it does not matter if LARAVEL_OCTANE is 1 or 0, because if it would be disabled (0) defined('LARAVEL_START') should be true anyway

Reply:
Added a function_exists check to satisfy larastan.
It feels a bit redundant, might only actually do something in strange situations where the laravel constant is defined but the framework is not included for real. Let me know if this is acceptable